contrib/docker/ckan-entrypoint.sh

```diff
@@ -58,29 +61,30 @@ link_redis_url () {
 
 # If we don't already have a config file, bootstrap
 if [ ! -e "$CONFIG" ]; then
+  write_config
+fi
```

Comment:
hmm, probably I misunderstand, but need to check. The case is, what if I need to change default values for connection string to redis and pg